src/System.Management.Automation/CoreCLR/EventResource.cs

```diff
@@ -0,0 +1,616 @@
+#if UNIX
+/*
+    This code was generated by the tools\ResxGen\ResxGen.ps1 run against PowerShell-Core-Instrumentation.man.
+    To add or change logged events and the associated resources, edit PowerShell-Core-Instrumentation.man
+    then rerun ResxGen.ps1 to produce an updated CS and Resx file.
+*/
+using System.Collections.Generic;
+using System.Management.Automation.Internal;
+using System.Runtime.InteropServices;
+
+namespace System.Management.Automation.Tracing
+{
+    /// <summary>
+    /// Provides a class for describing a message resource for an ETW event.
+    /// </summary>
+    internal static class EventResource
+    {
+        // Defines the resource id of the message to use when an event id is not valid.
+        const string MissingEventIdResourceName = "MissingEventIdMessage";
```

Comment:
Shouldn't it be `Private`?

Reply:
The pattern I see is to mark classes as internal versus private.  You will see the use of private for a class that is nested in quite a few places in the code base.

Reply:
I meant the const:
```c#
private const string MissingEventIdResourceName = "MissingEventIdMessage"; 
```